spec/features/user_can_be_deleted_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe "User view" do
+  context 'with valid attributes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.